packetery/src/Packetery/class-helper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Class Helper
+ *
+ * @package Packetery
+ */
+
+declare( strict_types=1 );
+
+
+namespace Packetery;
+
+/**
+ * Class Helper
+ *
+ * @package Packetery
+ */
+class Helper {
```

Comment:
V ostatních modulech máme na takové věci Tools, nebylo by dobré mít to jednotné?

Reply:
Třída Tools v PS řeší technické věci pro veškerou business logiku. Tato třída řeší sestavování trasovacího odkazu zásilky. Spíš by to asi mělo být v \Packetery\Packet\Service.  Zatím bych pojmenoval obecně, abychom neměli mini třídy na všechno.